### ISSUE TITLE
Keyboard navigation in menus [moved]

### ISSUE THREAD

**Issue by ooxi** [ooxi/violetland#42]
This is [Issue 42](http://code.google.com/p/violetland/issues/detail?id=42) moved from a Google Code project.
Added by 2010-09-18T20:35:11.000Z by [5253...@gmail.com](http://code.google.com/u/112832467119010425126/).
Please review that bug for more context and additional comments, but update this bug.
 Closed (WontFix).

Original labels: Type-Enhancement, Priority-Medium
### Original description

```
Game should have keyboard navigation in menus in additional to pointing mode.
```
